### ISSUE TITLE
Update the script to work with strongs numbers

### ISSUE THREAD

**Issue by scruffian** [danday74/sword-to-json#2]
I'm working on this, and will have a PR ready soon.

**Comment by TommyEsteban** [comment]
Yes, for some reason strong numbers are not processed. I've tested the script with this version
http://crosswire.org/sword/modules/ModInfo.jsp?modName=SpaRV1909

**Comment by scruffian** [comment]
Did you try this script: https://github.com/danday74/sword-to-json/pull/3 ?
